flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -358,6 +361,12 @@ public long getBufferTimeout() {
 		return this.bufferTimeout;
 	}
 
+
+	public StreamExecutionEnvironment setGlobalDataExchangeMode(GlobalDataExchangeMode globalDataExchangeMode) {
```

Comment:
I'd probably not expose `GlobalDataExchangeMode` directly, but rather go with `enableApproximateSinkFailover`.